### PR TITLE
UX Changes For P0 Components

### PR DESCRIPTION
**Pull request by varun13169**
UX Changes For P0 Components

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
Good to go Varun.!

File: components/alert/alertPage.css [hunk 1]
```diff
@@ -2,13 +2,18 @@
     display: flex;
     flex-direction: column;
     align-items: center;
+    padding-left: 20rem;
+    padding-right: 20rem;
```
Comment (humzahasan) [inline components/alert/alertPage.css:6]: padding : 0 20rem;  this might look better. Again personal choice.

**Review by humzahasan — APPROVED** [review]
Looks Good to merge!

File: components/alert/alertPage.css (hunk 1 above)
Comment (iRohitGaur) [reply components/alert/alertPage.css:6]: +1

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by zoef-yt — APPROVED** [review]
Looks go to me. Good to merge.

File: components/alert/alertPage.css (hunk 1 above)
Comment (varun13169) [reply components/alert/alertPage.css:6]: I am following the explicit way site wide, unless all 4 require change.
But yes I agree with you, it's just to maintain consistency.

**Pull request merged**